Makefile

```diff
@@ -44,10 +37,6 @@ ingestor-test-db-init:
 	docker compose exec db sh -c 'cat /tmp/sql/schema.sql | psql postgres://postgres:postgres@127.0.0.1:5432/cryoet'
 	docker compose exec db sh -c 'cat /tmp/sql/seed_db_enum.sql | psql postgres://postgres:postgres@127.0.0.1:5432/cryoet'
```

Comment:
I'm pretty sure we don't need this anymore. We also have some documentation files that refer to this that may need to be updated